updating/index.adoc

```diff
@@ -0,0 +1,90 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. Platform administrators are automatically notified when an update is available for their cluster. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
```

Comment:
> Platform administrators are automatically notified when an update is available for their cluster.
There seems not auto-notify. At least, there is not from CLI. @yanpzhan Do you happen to know if we have any notify from console if an update is available?I know we can check available update from `cluster settings`, but not sure if any automatic notify.  And @juzhao or any alert msg triggered for available update?

Reply:
I think it means if you configured upstream and channel correctly, you can see the available updates
1. on console, you can check via "Administration -> Cluster Settings" to see the available updates, and update to the version you wanted.
2. `oc adm upgrade` also indicate the available updates
3. There is one UpdateAvailable alert fired
```
# token=`oc sa get-token prometheus-k8s -n openshift-monitoring`
# oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -k -H "Authorization: Bearer $token" 'https://thanos-querier.openshift-monitoring.svc:9091/api/v1/query?query=ALERTS' | jq
...
      {
        "metric": {
          "__name__": "ALERTS",
          "alertname": "UpdateAvailable",
          "alertstate": "firing",
          "channel": "stable-4.10",
          "prometheus": "openshift-monitoring/k8s",
          "severity": "info",
          "upstream": "https://amd64.ocp.releases.ci.openshift.org/graph"
        },
        "value": [
          1645692086.968,
          "1"
        ]
      },
....
```

Reply:
Thanks! If so, i think we need correct the words here about `Platform administrators are automatically notified`. All above needed to be done with manually actions, but not an auto-notify.

Reply:
Addressed

Reply:
ack